grafana-formula/grafana-formula.changes

```diff
@@ -1,3 +1,5 @@
+  * Update dashboards
```

Comment:
Not sure if it would make sense mentioning why the update is needed? Would the following be relevant for customers?
```suggestion
  * Update dashboards to use the new JSON schema
```